### DISCUSSION TITLE
✅ 🔗 display conditions on Icon widgets (Duplicate of #4246)

### DISCUSSION THREAD

**Discussion by SpinDreams** [elementor/elementor#20050] [Feature Request]
It would be really useful to have the ability select to not to render the icon (Or the whole widget code) if there is no dynamic data returned in the title or description. Same for Icon lists and I am sure some other widgets. A great example is a property website which may have a bunch of icons with numbers next to them to indicate number of bedrooms, bathrooms or size in M2 etc, but if a property has no value for one of these the icon still renders and it looks really bad.

**Comment by SpinDreams** [comment]
Another option that could be used for lots of other options would be to have conditional containers to conditionally display a container and its content based on dynamic data values.  (Although this would not help for Icon lists items)

**Comment by nicholaszein** [comment]
Hello!

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.19.0, and improved with Elementor Pro 3.20.0! 🥳 

You can now use Display Conditions for widgets based on a variety of conditions, including date, time, user role, logged in/out, and others (more to come too). Check the [documentation for the new Display Conditions](https://elementor.com/help/show-and-hide-elements-on-a-page-display-conditions/) for widgets (Pro).

In your case you could check for the same dynamic data added in your title or description to show or hide the entire widget.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
